lib/alchemy/tasks/helpers.rb

```diff
@@ -48,19 +48,31 @@ def mysql_command(cmd = 'mysql')
 
       def postgres_command(cmd = 'psql')
         command = []
-        if database_config['password']
-          command << "PGPASSWORD='#{database_config['password']}'"
+
+        config = database_config
```

Comment:
Not sure why you need this? Could just use `database_config` where `config` is being used?